src/core/packages/chrome/navigation/src/__tests__/test_setup.ts

```diff
@@ -46,3 +46,8 @@ afterAll(() => {
 
   jest.clearAllMocks();
 });
+
+jest.mock('../hooks/use_raf_debounced', () =>
+  // eslint-disable-next-line @kbn/imports/no_boundary_crossing
+  jest.requireActual('../hooks/use_raf_debounced.mock')
```

Comment:
@weronikaolejniczak I thought this is ok to keep tests simpler and not make them even slower. otherwise need to add couple waitFor's here and there. but can change if you'd prefer